test/fixtures/specs/broken.js

```diff
@@ -3,7 +3,7 @@
 describe('A broken Suite', () => {
     it('with broken test', () => {
         return browser
-          .url('/index.html')
-          .click('.missing-element')
+            .url('/index.html')
+            .missingCommand()
```

Comment:
Here another "hack". When `.click('.missing-element')` performs it doesn't mark step as failed or broken because there are no not finished steps(actually in test xml only runner steps with json requests\responses).